src/app/lend/pages/lend-landing/lend-landing.page.ts

```diff
@@ -15,68 +15,75 @@ import { LendingPool, Loan, LoanRequest } from '../../lend.models';
 })
 export class LendLandingPage implements OnInit, OnDestroy {
 
-  @ViewChild('segmentControl') segmentControl: IonSegment
+  @ViewChild('segmentControl') segmentControl: IonSegment;
 
-  loanRequests: LoanRequest[] = []
-  currentSegment = 'requests'
-  currency
-  openLoansSubscription?: Subscription
-  openLoanRequestsSubscription?: Subscription
-  lendingPools: LendingPool[] = []
-  vaultContract
-  ratePrecision = 100000
-  utilizationPrecision = BigNumber.from(1000000000000)
-  aprPrecision = BigNumber.from(100000000000)
+  loanRequests: LoanRequest[] = [];
+  currentSegment = 'requests';
+  currency;
+  openLoansSubscription?: Subscription;
+  openLoanRequestsSubscription?: Subscription;
+  lendingPools: LendingPool[] = [];
+  vaultContract;
+  ratePrecision = 100000;
+  utilizationPrecision = BigNumber.from(1000000000000);
+  aprPrecision = BigNumber.from(100000000000);
 
   constructor(
     private wallet: WalletProviderService,
     private priceFeed: PriceOracleService,
-    private vaultService: CheddaBaseTokenVaultService) { }
+    private vaultService: CheddaBaseTokenVaultService
```

Comment:
Create a VaultStatsService that has the `loadVaultStats()` function, and on init set the local `lendingPools` to `statsService.lendingPools`. On any change, call `loadVaultStats` in service.